src/App/Pages/LoginPage.cs

```diff
@@ -112,14 +112,20 @@ private void Init()
                 }));
             }
 
-            var loginToolbarItem = new ToolbarItem(AppResources.LogIn, null, async () =>
+            var loginToolbarItem = new ToolbarItem(AppResources.LogIn, Helpers.LoginImage(), async () =>
             {
                 await LogIn();
             }, ToolbarItemOrder.Default, 0);
 
             ToolbarItems.Add(loginToolbarItem);
             Title = AppResources.Bitwarden;
-            Content = scrollView;
+            if (Device.RuntimePlatform == Device.Windows)
```

Comment:
Why was this needed?

Reply:
The scroll ends up being really weird without this. Instead of everything fitting in a single page (which it could) it gets expanded to scroll. The register page is even worse. Just run the UWP app and you will see the issue

Reply:
We likely need to do some work on the custom renderers for TableView like we have for the other platforms.

Reply:
Yeah, I think that is the problem. I looked at the vault last night and that display is all messed up.

Where is a good place to start?